logstreams/src/test/java/io/camunda/zeebe/logstreams/log/LogStreamBatchWriterTest.java

```diff
@@ -236,27 +228,6 @@ public void shouldWriteEventWithoutSourceEvent() {
     assertThat(events.get(1).getSourceEventPosition()).isEqualTo(-1L);
   }
 
-  @Test
-  public void shouldWriteEventWithTimestamp() throws InterruptedException, ExecutionException {
```

Comment:
:thought_balloon: I don't know if we should remove it from the API tests. In the end, it's part of the contract that if I write something, when I read it back, the timestamp will be there, no? I understand there's some overlap with the serialization process, but that's implementation details, I think.

Reply:
I saw those tests more as testing the write API of the logstream module as a whole, not the specific writer. But I guess they were sort of specifically testing this interface :smile: 

All good :+1: